source/common/ssl/openssl.h

```diff
@@ -4,25 +4,21 @@
 
 #include "openssl/ssl.h"
 
+#ifndef OPENSSL_IS_BORINGSSL
+#error Envoy requires BoringSSL
+#endif
+
 namespace Ssl {
 
 /**
- * Global functionality specific to OpenSSL.
+ * Global functionality specific to BoringSSL.
  */
 class OpenSsl final {
```

Comment:
Actually, upon further investigation, it looks that we don't need to call `SSL_library_init()` at all, which means that `Ssl::OpenSsl` can be removed.